compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BTupleType.java

```diff
@@ -89,7 +90,7 @@ private BTupleType(BTypeSymbol tsymbol, boolean readonly) {
     }
 
     @Override
-    public List<BType> getTupleTypes() {
+    public List<BTupleMember> getTupleTypes() {
```

Comment:
Just to get an idea, how often did we need to access the annotations + the types vs access just the types. Wondering if it may make sense to leave this method as is (this will return just a list of `BType`s) and introduce a separate `getTupleMembers` method that will actually return the list of `BTupleMember`s.

Reply:
Currently it's mostly used just for the types when considering individual members, I'll change as suggested